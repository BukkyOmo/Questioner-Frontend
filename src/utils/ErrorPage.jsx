import React from 'react';

const ErrorPage = () => <h1>Error 404: Not found</h1>;

export default ErrorPage;
