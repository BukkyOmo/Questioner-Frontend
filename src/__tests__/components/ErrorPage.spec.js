import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from '../../utils/ErrorPage';

describe('<ErrorPage />', () => {
  it('renders h1 element', () => {
    const wrapper = shallow(<ErrorPage />)
    expect(wrapper.find('h1').exists()).toBe(true);
  });
   it('renders correctly using snapshot', () => {
    expect(ErrorPage).toMatchSnapshot();
   });
});
