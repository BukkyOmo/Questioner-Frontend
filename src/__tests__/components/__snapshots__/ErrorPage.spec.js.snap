// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage /> renders correctly using snapshot 1`] = `[Function]`;
